lib/ref_arch_setup/install.rb

```diff
@@ -35,17 +35,54 @@ def initialize(target_master)
     # @param [string] pe_conf_path Path to pe.conf
     # @param [string] pe_tarball Path or URL for the pe tarball
     #
+    # @raise [RuntimeError] Unless either a pe_tarball or pe_version is specified
+    # @raise [RuntimeError] If a valid pe_tarball is not available
+    # @raise [RuntimeError] If a RAS working directory can not be created
+    #
     # @return [true,false] Based on exit status of the bolt task
-    def bootstrap(pe_conf_path, pe_tarball)
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/AbcSize
+    def bootstrap(pe_conf_path, pe_tarball, pe_version)
+      if specified_option?(pe_tarball)
+        puts "Proceeding with specified pe_tarball: #{pe_tarball}"
+        @pe_tarball = pe_tarball
+      else
+        options_error = "Either a pe_version or pe_tarball must be specified"
+        raise options_error unless specified_option?(pe_version)
+        puts "Proceeding with specified pe_version: #{pe_version}"
+        @pe_tarball = RefArchSetup::DownloadHelper.build_prod_tarball_url(pe_version,
+                                                                          @target_master)
+      end
+
+      raise "Unable to proceed without pe_tarball" unless @pe_tarball
```

Comment:
In what case would this actually get triggered?  does the message communicate that?

Reply:
Removed as it should not be possible for this to be triggered